config/config.toml

```diff
@@ -61,6 +61,8 @@ LogSpecialRecords = true
 PprofPort = 6000
 # Prometheus port
 PromPort = 9090
+# Switch to expose only small subset of essential metrics
+ShortMetrics = false
```

Comment:
ShortMetrics is ambiguous. 
Proposing: LessMetrics
Also update the comment with a line explaining why: "Useful if running as a sidecar on many nodes across a big setup"

Reply:
Did that!